xml/MTL/3/mtl_aggregator.xml

```diff
@@ -49,15 +49,15 @@
 	<TELEM:SampleGroup name="Container_2" sampleID="2" sampleGroupID="Container_2">
 		<TELC:description>Groupname 0x16010</TELC:description>
 		<TELC:length>64</TELC:length>
-		<TELC:sample name="DIE_CSTATE_BLOCK_CAUSE_CORE" datatypeIDREF="tpkgc_block_cause" sampleID="DIE_CSTATE_BLOCK_CAUSE_CORE">
```

Comment:
nit. `datatypeIDREF` for the `sample` does not exactly fit. I suppose our XSD could use use some predefined *primitive* types in case `sample` on it's own doesn't have any specific type or we just don't need to disclose the actual meaning in XML.

Reply:
good plan